### PR TITLE
Code complete with setting property in Nodes

### PR DESCRIPTION
**Pull request by ilooner**

### CODE REVIEW

**Comment by ilooner** [comment]
@PramodSSImmaneni I am code complete with setting properties in Nodes. I still need to write unit tests.

File: bufferserver/src/main/java/com/datatorrent/bufferserver/packet/MessageType.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
  */
 public enum MessageType
 {
-  NO_MESSAGE(0), PAYLOAD(1), RESET_WINDOW(2), BEGIN_WINDOW(3), END_WINDOW(4), END_STREAM(5), PUBLISHER_REQUEST(6), SUBSCRIBER_REQUEST(7), PURGE_REQUEST(8), RESET_REQUEST(9), CHECKPOINT(10), CODEC_STATE(11), NO_MESSAGE_ODD(127);
+  NO_MESSAGE(0), PAYLOAD(1), RESET_WINDOW(2), BEGIN_WINDOW(3), END_WINDOW(4), END_STREAM(5), PUBLISHER_REQUEST(6), SUBSCRIBER_REQUEST(7), PURGE_REQUEST(8), RESET_REQUEST(9), CHECKPOINT(10), CODEC_STATE(11), MODIFY_CONFIGURATION(12), NO_MESSAGE_ODD(127);
```
Comment (pramodin) [inline bufferserver/src/main/java/com/datatorrent/bufferserver/packet/MessageType.java:28]: This type has already been added in an earlier commit

File: bufferserver/src/main/java/com/datatorrent/bufferserver/packet/MessageType.java [hunk 2]
```diff
@@ -38,6 +38,7 @@
   public static final byte RESET_REQUEST_VALUE = 9;
   public static final byte CHECKPOINT_VALUE = 10;
   public static final byte CODEC_STATE_VALUE = 11;
+  public static final byte MODIFY_CONFIGURATION_VALUE = 12;
```
Comment (pramodin) [inline bufferserver/src/main/java/com/datatorrent/bufferserver/packet/MessageType.java:41]: I already have these in the local file. Could you backout the changes in this file only.

**Comment by pramodin** [comment]
Can you take a look at what I have in the branch and merge your changes. I was using the same container for stram ->operator  and the same would be used for operator -> operator. I am hoping that the same container can be used for CLI -> Stram and may require some minor changes.

**Comment by pramodin** [comment]
When you merge and do pull request can you move your changes into ModifyConfiguration payload in stram api and we don't have to use operator request like I have. You can leave the StreamingContainerManager as is with errors or comment out the place it is setting the operator request in ModifyConfiguration. After merging your request I can change it to use PropertyChange.

**Comment by pramodin** [comment]
Can you move ConfigurationChangeBatch and its encapsulated classes to com.datatorrent.stram.api so that I can merge ModifyConfigurationTuple cleanly.

**Comment by ilooner** [comment]
@PramodSSImmaneni ConfigurationChangeBatch and associated classes into stram.api

**Pull request merged**